conjur/logic/init_logic.py

```diff
@@ -42,8 +45,17 @@ def get_certificate(self, hostname:str, port):
         try:
             fingerprint, readable_certificate = self.ssl_service.get_certificate(hostname, port)
             logging.debug("Successfully fetched certificate")
+        except socket.gaierror as error:
```

Comment:
I think that according to @oburstein-hub coding standarts we should import only the error if we are not using socket

`from socket import gaierror as SocketGetAddressInfoException, timeout as SocketTimeoutException`

Reply:
done